src/Mvc/Mvc.ApiExplorer/src/EndpointMetadataApiDescriptionProvider.cs

```diff
@@ -124,6 +124,8 @@ private ApiDescription CreateApiDescription(RouteEndpoint routeEndpoint, string
 
             // Get custom attributes for the handler. ConsumesAttribute is one of the examples. 
             var acceptsRequestType = routeEndpoint.Metadata.GetMetadata<IAcceptsMetadata>()?.RequestType;
+            var isRequired = routeEndpoint.Metadata.GetMetadata<IAcceptsMetadata>()?.IsRequired;
```

Comment:
We should be resolving this once:

```suggestion
            var acceptsMetadata = routeEndpoint.Metadata.GetMetadata<IAcceptsMetadata>();
            var acceptsRequestType = acceptsMetadata?.RequestType;
            var isRequired = acceptsMetadata?.IsRequired;
```